moto/utilities/Paginator.py

```diff
@@ -0,0 +1,139 @@
+from functools import wraps
```

Comment:
Any particular reason why this filename is capitalized?

Reply:
Oversight - thanks for pointing it out